src/components/tabs/README.md

```diff
@@ -1 +1,141 @@
+    </div>
+  <div class="rvt-tabs__panel" tabindex="0" role="tabpanel" id="tab-3" aria-labelledby="t-three" data-tab-panel="tab-3" hidden data-tab-init>
+    <span class="rvt-ts-23 rvt-text-bold">A grid inside a tab panel</span>
+    <p>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+  </div>
+</div>
+```
+
+## Initialization
+
+In order to initialize the modal as an ES6 module, you will need to create a new instance of it, and pass it the element you wish to instantiate as a set of tabs.
+
+```
+const tabsElement = document.querySelector('[data-tabset="tabset-1"]');
+document.newTabs = new Rivet.Tabs(tabsElement);
+```
+
+## "Public" methods
+
+If you use the appropriate data attribute/id combination in your markup, modals will work without the need for any additional JavaScript. But if you need to control the modal programmatically, there are a handful of methods from the Rivet modal’s API you can use:
+
+| Method                      | Description                                                                                                                      |
+| --------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| .activateTab(tab, callback) | - `tab` - The element to be activated. - `callback` - An optional callback function that is executed after the tab is activated. |
+| .destroy()                  | Adds the built-in event listeners to the tab component.                                                                          |
+| .init()                     | Removes all built-in event listeners from the tab component.                                                                     |
+
+## Custom events
+
+| Event          | Description                                                                                                                                                                                                                                                                                                                        |
+| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `tabActivated` | Emitted when an individual tab panel is activated (using the `Tabs.activateTab()` method, or via a click on a button with the `data-tab` attribute). The value of the `data-tab` attribute is also passed along (if it exists) via the custom event’s detail property and is available to use in your scripts as event.detail.id() |
```

Comment:
I believe in Rivet 1.x.x javascript the `event.detail.id` was a function that returned the id, which is why the old docs where written `event.detail.id()` (calling the function). With the new JS, we have simplified it so it's a plain property on the `event.detail` object.
```suggestion
| `tabActivated` | Emitted when an individual tab panel is activated (using the `Tabs.activateTab()` method, or via a click on a button with the `data-tab` attribute). The value of the `data-tab` attribute is also passed along (if it exists) via the custom event’s detail property and is available to use in your scripts as event.detail.id |
```